### ISSUE TITLE
eXIf Exchangeable Image File (Exif) Profile

### ISSUE THREAD

**Issue by svgeesus** [w3c/PNG-spec#28]
The [eXIf Exchangeable Image File (Exif) Profile](http://ftp-osl.osuosl.org/pub/libpng/documents/pngext-1.5.0.html#C.eXIf) chunk should probably be added to the main PNG specification, rather than being off to the side in an extensions document.

**Comment by svgeesus** [comment]
Raised by Phil Harvey <phil@owl.phy.queensu.ca>

> Although the original eXIf specification didn't mandate a location for the eXIf chunk, 
> web developers are advising to ignore the eXIf if it comes after IDAT:

> https://github.com/w3c/csswg-drafts/issues/4929

> As such, you may want to consider amending the specification to reflect this.  
> Minimally, it should be recommend to place eXIf before IDAT for web compatibility.  
> Or possibly requiring it to come before IDAT.

**Comment by svgeesus** [comment]
Pull request https://github.com/w3c/PNG-spec/pull/81

**Comment by svgeesus** [comment]
> As such, you may want to consider amending the specification to reflect this.
Minimally, it should be recommend to place eXIf before IDAT for web compatibility.
Or possibly requiring it to come before IDAT.

Note that the above PR just merges in the old text from the extensions document.

I agree that this restriction should be added, but plan to do that as a separate change.

**Comment by ProgramMax** [comment]
With pull request #81 merged, can this issue be closed?

Should the separate change for the restriction also have a separate issue?

**Comment by svgeesus** [comment]
Yes this is [all in the spec now](https://w3c.github.io/PNG-spec/#eXIf)
